Makefile

```diff
@@ -32,10 +35,9 @@ check-fmt: fmt
 test:
 	go test $(GOFLAGS) ./...
 
-.PHONY: build-push
-build-push:
-	hack/app_sre_build_push.sh $(IMAGE_URI_VERSION)
-
 .PHONY: boilerplate-update
 boilerplate-update:
 	@boilerplate/update
+
+
```

Comment:
Can you please cleanup the extra new lines ?

Reply:
done